### PR TITLE
Added some comments and tried to add a border to the field.

### PR DESCRIPTION
**Pull request by TBlair256**
Taylor first pull test

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -137,8 +138,12 @@ public void autonomousInit() {
       new Turn(m_hardware, m_robotDrive, turnDegrees),
       new Drive(m_hardware, m_robotDrive, inches(starToMiddle)),
       new Turn(m_hardware, m_robotDrive, turnDegrees),
-      new Drive(m_hardware, m_robotDrive, inches(backHome))
+      new Drive(m_hardware, m_robotDrive, inches(backHome)),
+      new Turn(m_hardware, m_robotDrive, turnDegrees),
+      new Turn(m_hardware, m_robotDrive, turnDegrees),
+      new Turn(m_hardware, m_robotDrive, turnDegrees)
       );
+      //somehow I changed something, distances seem to have been changed.
```
Comment (fixermark) [inline src/main/java/frc/robot/Robot.java:146]: Can you clarify what this means?

Comment (TBlair256) [reply src/main/java/frc/robot/Robot.java:146]: I copy and pasted the code used for the border from the project where it worked, but it seems to have caused the robot to not drive as far as it should...

File: src/main/java/frc/robot/PoseEstimator.java [hunk 2]
```diff
@@ -71,6 +71,13 @@ public void updatePose(Hardware hardware) {
         m_pose = new Pose2d(
```
Comment (fixermark) [inline src/main/java/frc/robot/PoseEstimator.java]:
```suggestion
        newX=Math.max( 0, newX);
        newX=Math.min(16.4846, newX);
        newY=Math.max( 0, newY);
        newY=Math.min(8.1026, newY);

        m_pose = new Pose2d(
```

File: src/main/java/frc/robot/PoseEstimator.java [hunk 3]
```diff
@@ -71,6 +71,13 @@ public void updatePose(Hardware hardware) {
...
             new Translation2d(newX, newY), 
             Rotation2d.fromDegrees(-hardware.gyro().getAngleZ()));
+/*
```
Comment (fixermark) [inline src/main/java/frc/robot/PoseEstimator.java]: Go ahead and remove this comment.

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (fixermark) [reply src/main/java/frc/robot/Robot.java:146]: Cool. Let's figure that part out next.

File: src/main/java/frc/robot/SimpleSimulatedChassis.java [hunk 4]
```diff
@@ -52,6 +56,19 @@ public SimpleSimulatedChassis (RomiGyro gyro, SimulatedEncoder leftEncoder, Simu
         m_leftEncoder = leftEncoder;
         m_rightEncoder = rightEncoder;
         m_lastUpdateTime = Timer.getFPGATimestamp();
+
```
Comment (fixermark) [inline src/main/java/frc/robot/SimpleSimulatedChassis.java:59]: This code is up in the constructor; you can remove these comments.

**Review by fixermark — CHANGES_REQUESTED** [review]

**Review by fixermark — APPROVED** [review]

**Pull request merged**